output.json

```diff
@@ -11,6 +11,10 @@
         {
             "invoke": "buttontokens.pr",
             "write_to": "DesignSystem/Components/Button/ButtonTokens.swift"
+        },
+        {
+            "invoke": "primarybutton.pr",
+            "write_to": "DesignSystem/Components/Button/PrimaryButton.swift"
```

Comment:
I think you and @lysenkoj talked about it in the open house, but I think we shouldn't continue expecting to manually add a file for each component; it should be automatically generated/emitted for each component. You can take a look at my PR for an example for how it can be accomplished.

Reply:
Yeah, that would be best! Especially as our list of components grow. Where exactly did you setup the automatic distribution in the android exporter? I wasn't able to locate it.

Reply:
Yeah this would be nice. I do think this is something we can circle back to in the future though